charts/mesh-v2-test-suite/README.md

```
@@ -0,0 +1,148 @@
+| bookinfo-ratings-v1     | `asm/istio/examples-bookinfo-ratings-v1`     | `1.20.3`  |
+| bookinfo-reviews-v1     | `asm/istio/examples-bookinfo-reviews-v1`     | `1.20.3`  |
+| bookinfo-reviews-v2     | `asm/istio/examples-bookinfo-reviews-v2`     | `1.20.3`  |
+| bookinfo-reviews-v3     | `asm/istio/examples-bookinfo-reviews-v3`     | `1.20.3`  |
+| bookinfo-productpage-v1 | `asm/istio/examples-bookinfo-productpage-v1` | `1.20.3`  |
+| tcp-echo-server         | `asm/istio/tcp-echo-server`                  | `1.3`     |
+
+镜像源 registry 由 `global.registry.address` 配置，默认 `build-harbor.alauda.cn`。
+
+## 目录结构
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)